Dockerfile

```
@@ -0,0 +1,23 @@
+RUN yarn install
+
+# Copy the source. (node_modules are excluded in .dockerignore)
+COPY survey survey
+
+# Setup the example as a default configuration for the image
+COPY .env.example /app/.env
+
+RUN yarn compile
+
+# Start Node app
+CMD yarn build:prod && yarn start
```

Hadolint (2.14.0)

[warning] Use arguments JSON notation for CMD and ENTRYPOINT arguments

(DL3025)